examples/celery-rabbitmq/flower-controller.yaml

```diff
@@ -18,3 +18,10 @@ spec:
         resources:
           limits:
             cpu: 100m
+        livenessProbe:
+          httpGet:
+            # Path to probe; should be cheap, but representative of typical behavior
+            path: /
+            port: 80
+          initialDelaySeconds: 30
+          timeoutSeconds: 1
```

Comment:
I wasn't sure if a cluster under load might result in this taking more than 1 second. it looks like many of our testing manifests use 5s, though I do see a few other 1s timeouts scattered throughout. not a big deal in any case.